src/Woz.BevragenProxy.Specifications/BepaalRelevanteWaarden.feature

```diff
@@ -0,0 +1,180 @@
+﻿#language: nl
+Functionaliteit: Bepaal relevante waarden voor een WOZ-object
+
+Scenario: geen waarden
+	Gegeven een WOZ-object bevat geen waarden
+	Als de relevante waarden zijn bepaald voor het WOZ-object
+	Dan bevat het WOZ-object geen waarden
+
+Scenario: leeg waarden lijst
+	Gegeven een WOZ-object bevat de volgende waarden
+	| vastgesteldeWaarde | waardepeildatum | ingangsdatum | beschikkingsStatussen |
```

Comment:
hoe moet ik me deze voorstellen? De tabel heeft geen rijen. Bedoel je dan een lege collectie waarden?
`"waarden": []`

Dat is toch een foutsituatie? Want als er geen waarden zijn, krijg je ook geen property "waarden". En dat is het vorige scenario.

Reply:
Klopt, maar technisch kan het wel. Het wordt namelijk niet afgedwongen door de specificatie dat een lege collectie niet mag. Dit betekent dat bijv. een API gateway de waarden kan filteren en ipv daarvan een lege collectie terug kan geven. Daarom is deze scenario toegevoegd.

Reply:
is het dan niet duidelijker om aan te geven "een lege collectie waarden" of "waarden": [], in plaats van een tabel zonder rijen?

Reply:
done